package-lock.json

```diff
@@ -1,15530 +1,8 @@
 {
   "name": "@pocket/terraform-modules",
   "version": "0.0.0-development",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@bkochendorfer, I think you need to regenerate this file with a different version of `npm` (v. 7 that comes with Node 16?). The easiest way I've found is to, after making sure you're on the correct version of Node, delete the `package-lock.json` file and run `npm install --package-lock-only`.